vsts/pipelines/_buildTemplate.yml

```diff
@@ -5,7 +5,7 @@ steps:
     pipelineId: 'Oryx-CI'
     artifactName: 'drop-$(Build.BuildNumber)'
     targetPath: '$(Build.SourcesDirectory)'
-  condition: and(succeeded(), eq(variables['Build.DefinitionName'], 'Oryx-CI'), in(variables['Build.Reason'], 'Schedule', 'Manual'))
```

Comment:
Just FYI...make sure to have proper message on the PR as that is what is being put as the main commit message. For example, from earlier PR of yours:
![image](https://user-images.githubusercontent.com/6559784/54843706-477fba00-4c92-11e9-98ae-00bd1024b9ea.png)